Makefile

```diff
@@ -9,6 +9,9 @@ TEST_PATH ?= .
 PYTEST_LOGLEVEL ?= warning
 MAIN_CONTAINER_NAME ?= localstack_main
 
+MAJOR_VERSION = $(shell echo ${IMAGE_TAG} | cut -d '.' -f1)
+MINOR_VERSION = $(shell echo ${IMAGE_TAG} | cut -d '.' -f2)
```

Comment:
Unfortunately we currently have four-segment versions. For example, the current version is `0.13.0.10`.
With this fourth version segment, we would also need to make sure to update the third segment in case the fourth segment is updated. So basically we need to update `0.13.0` when we release `0.13.0.10`.
Unfortunately, it's currently not safe to assume that the fourth digit is always there.
This might complicate things a bit...

Reply:
Hmm, let me check to see if I understood what you mean.
So you want to have a tag for both minor version (`0.13`) and patch version (`0.13.0`)?
Currently as It's implemented, if you do a release `0.13.0.11`, the docker will tag and push the new image with `0.13` so it will update the tag to include the new changes.